README.md

```diff
@@ -1,70 +1,153 @@
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+### Key Features <a name="key-features"></a>
 
-### Analyzing the Bundle Size
+- **Linter for React**
+- **LInter for CSS**
+- **Eslint**
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+<!-- GETTING STARTED -->
 
-### Making a Progressive Web App
+## 💻 Getting Started <a name="getting-started"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+To get a local copy up and running, follow these steps.
 
-### Advanced Configuration
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+In order to run this project you need:
+<ul>
+    <li>Web Browser</li>
+     <li>Node.js installed</li>
+     <li>Code Editor CLI</li>
+  </ul>
 
-### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+### Setup
 
-### `npm run build` fails to minify
+Clone this repository to your desired folder:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+```sh
+  cd desired folder
+  git clone https://github.com/donice/Portfolio-Microverse.git
+  cd metrics
+  run npm install
+  run npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+run npm test
+```
+
+```sh
+  npx npx eslint "**/*.{js,jsx}"
+  npx stylelint "**/*.{css/scss}"
+```
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/paulshaun-kdtk)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shaun-sungai-kdtk/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team.
+As well as Nelson Sakwa; the owner of the original template used for the design of this website.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] 👋 Hello! You've taken an important step by adding a license to your project, which is great to see! 😄 However, there's a minor issue with the license link in your project. It seems that you're referencing a file called MIT.md in the root of the project, but this file does not currently exist. The actual license information is stored in a LICENSE file in the root folder.

To address this, you have two options:

1. Update the License Link:

- In your project's README or documentation, update the license link to point to the existing LICENSE file.

For example, you can modify the license section in your README like this:
```sh
### License
This project is licensed under the [MIT License](./LICENSE).
```

2. Create an MIT.md File:

- If you prefer to have a separate MIT.md file, you can create one in the root of your project and move the license information there. You can then update the license link accordingly.

To create an MIT.md file and move the license information, you can use the following commands in your project's root directory:

```sh
mv LICENSE MIT.md
```

By making these adjustments, you'll ensure that your project's license information is accurately linked and easily accessible to others. If you have any questions or need further assistance, feel free to ask! 📜👩‍💻🚀